appveyor.yml

```diff
@@ -0,0 +1,32 @@
+environment:
+  matrix:
+  - PYTHON: C:\\Python35-x64
+clone_depth: 1
+cache:
+- '%LOCALAPPDATA%\pip\Cache\http'
+- '%LOCALAPPDATA%\pip\Cache\wheels'
+build: off
+before_test:
+- cmd: |
+    %PYTHON%\\python.exe -m venv venv
+    venv\\Scripts\\activate.bat
+    ECHO %VIRTUAL_ENV%
+    python -c "import sys; print(sys.executable)"
+    python -m pip install -r requirements.txt
+    python -m pip install -r requirements-devel.txt
+    python setup.py install
```

Comment:
this might be better as `python -m pip install .` or does that not work on windows?

Reply:
It didn't seem to work when I first tried it. Apparently with `--prefix` it does. Unfortunately we're not getting an `.exe` this way so I need to execute Snapcraft in the tests via `python -m` now.